clients/include/testing_gemm.hpp

```diff
@@ -163,7 +153,7 @@ void testing_gemm(const Arguments& arg)
         rocblas_seedrand();
         rocblas_init<T>(hA, A_row, A_col, lda);
         rocblas_init_alternating_sign<T>(hB, B_row, B_col, ldb);
-        if(rocblas_isnan(arg.beta))
+        if(rocblas_isnan(arg.get_beta<T>()))
```

Comment:
`rocblas_isnan()` should be applied to `arg.beta` rather than `arg.get_beta<T>()`.

Reply:
understand I will correct it on third commit "update complex change".  But what if beta is not nan but betai is nan??  Do we have to consider this case??

Reply:
`is_nan` should be defined for `rocblas_float_complex` and `rocblas_double_complex`, which will return `true` if either the real or imaginary part is `NaN`.

Reply:
But since `arg.beta` and `arg.betai` are always floating-point types, you can use `std::isnan(arg.beta) || std::isnan(arg.betai)`.

Reply:
understand. update it in commit "update complex change 2".